CITATION.cff

```diff
@@ -0,0 +1,73 @@
+cff-version: 1.2.0
+message: If you use this software, please cite it as below.
```

Comment:
This is for human consumption and could for example also include instructions how to generate the bibtex entry.